### PR TITLE
docs: description of the project

### PR DESCRIPTION
**Pull request by felix-lambert**
I did'nt go too far because the doc of Julien might be included as a link to this Readme

### CODE REVIEW

**Review by lutangar — APPROVED** [review]
This is far beyond my expectations, love it!

Could just provide a generic contributing guide? This can be done later tho, I'm approving this :+1:

**Comment by felix-lambert** [comment]
Cool for this PR I will not merge before everyone approves :)

File: README.md
Comment (felix-lambert) [reply README.md:60]: Should I do the same for the backend readme and the extension readme?

Comment (lutangar) [reply README.md:60]: > Are you talking about the updates you mades on backend *contributing*?

About having a generic `CONTRIBUTING.md` for all repositories I would love to <3
Not sure what's the best way to implements it tho, for example:
- Shall we have a specific `CONTRIBUTING.md` in each repository - when required - and link to the generic one?
Or
- Shall we leverage the [`.github` mechanism](https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file#supported-file-types) and have the same `CONTRIBUTING.md` for all repositories and add all the specificities of a project in another `md` (`README.md`, suggestions?) file? 
cc @dis-moi/devs

Comment (felix-lambert) [reply README.md:60]: By intuition, I'm more tempted by your second question (.github mechanism)

Comment (christpet) [reply README.md]: @felix-lambert no need, just for this repo is good!

File: README.md
Comment (christpet) [inline README.md:7]:
```suggestion
To learn more about the use cases, visit the :fr: [**Dismoi** website](https://www.dismoi.io/).
```

**Review by christpet — APPROVED** [review]
Fantastic work @felix-lambert ! I say merge it :) then we can share with the team!

**Pull request merged**
